src/models/community_business.ts

```diff
@@ -602,6 +603,27 @@ export const CommunityBusinesses: CommunityBusinessCollection & CustomMethods =
             } , {});
           return { age };
         }),
+      lastWeek: applyQueryModifiers(client('visit')
```

Comment:
Why is this an aggregate of its own? Why not just use the `whereBetween` options on existing aggregates.

Reply:
Each aggregate is counting visits split up into certain categories, with this one splitting up the result into days of the week. This query could use the `CBs.getVisitLogs` method with a whereBetween, but would then need post processing display the data correctly ie:
```
  lastWeek: {
    '2018-09-05': 4,
    '2018-09-06': 2,
    '2018-09-07': 0 ...
  }
```

For readability, this method is defining the query itself.